File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(i int) (s string) {
	hasFactor := 0
	if i % 3 == 0 {
		s += "Pling"
		hasFactor++
	}
	if i % 5 == 0 {
		s += "Plang"
		hasFactor++
	}
	if i % 7 == 0 {
		s += "Plong"
		hasFactor++
	}
	if hasFactor == 0 {
		s = strconv.Itoa(i)
	}
	return
}

File: python/raindrops/raindrops.py
def convert(number):
    FACTORS_TO_SOUNDS = {
        3: "Pling"
        }
    result = ""
    if number % 3 == 0:
        result += "Pling"
    if number % 5 == 0:
        result += "Plang"
    if number % 7 == 0:
        result += "Plong"
    if number % 3 != 0 and number % 5 != 0 and number % 7 != 0:
        result += str(number)
    return result
